src/dispatch/plugins/dispatch_slack/events.py

```diff
@@ -319,3 +324,34 @@ def ban_threads_warning(
             message,
             thread_ts=event.event.thread_ts,
         )
+
+
+@slack_background_task
+def message_tagging(
+    user_id: str,
+    user_email: str,
+    channel_id: str,
+    incident_id: int,
+    event: EventEnvelope = None,
+    db_session=None,
+    slack_client=None,
+):
+    """Looks for incident tags in incident messages."""
+    text = event.event.text
+    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
+    tags = tag_service.get_all(db_session=db_session, project_id=incident.project.id).all()
+    log.debug(f"Fetched {len(tags)} tags from database.")
+
```

Comment:
I'd remove the `log.debug`. It may cause the server logs to grow considerably given the amount of times the function will run.
```suggestion
```